libs/services/designer-client-services/src/lib/standard/connector.ts

```diff
@@ -8,7 +8,9 @@ export class StandardConnectorService extends BaseConnectorService {
     operationId: string,
     _parameterAlias: string | undefined,
     parameters: Record<string, any>,
-    dynamicState: any
+    dynamicState: any,
+    nodeInputs: any,
+    nodeMetadata: any
```

Comment:
Is it possible to strongly type these? (We are trying to avoid use of `any` wherever possible, even in interface implementations.)

Reply:
I'm going to save these for a later PR